front/components/assistant/conversation/input_bar/InputBarContainer.tsx

```diff
@@ -269,8 +280,26 @@ const InputBarContainer = ({
     [editorRef, fileUploaderService, sendNotification]
   );
 
+  // Wrap onEnterKeyDown to clear draft when editor is cleared.
+  const wrappedOnEnterKeyDown: CustomEditorProps["onEnterKeyDown"] =
```

Comment:
We need to clear it too when you click the send button instead of pressing enter 😄 (it seems to be working for existing conversations but not new

Reply:
cleaned up to put it in the central onSubmit button